packages/shapes/src/utils/join-points.ts

```diff
@@ -52,11 +52,12 @@ export const joinPoints = (
 					];
 				}
 
+				const shorten = shortenVector(nextPoint, cornerRadius)
 				return [
 					{
 						type: 'M',
-						x,
-						y,
+						x: shorten[0],
+						y: shorten[1],
```

Comment:
I think this is erroneous, `x` and `y` are the coordinates of the current point, while `shorten` is being calculated based off the next point 🤔